pkg/util/image_prepper.go

```diff
@@ -54,17 +54,23 @@ func getImage(prepper Prepper, source string) (Image, error) {
 	}, nil
 }
 
-func (p ImagePrepper) GetImage() (Image, error) {
+func (p *ImagePrepper) GetImage() (Image, error) {
```

Comment:
Its weird that GetImage() actually mutates the fields on the struct.  Calling this twice on the same imagePrepper will break, right?  I would refactor this.  getImage already takes in redundant information since it takes `prepper` and `prepper.ImagePrepper.Source`

Reply:
the reason I changed it to do this is to handle the prefixes that the image prepper respects to force a certain source (e.g. `daemon://<image_path>`). these need to be removed before actually trying to parse the image reference with the containers/image library. `GetImage()` only ever gets called one time per prepper, so there shouldn't be any risk here. `getImage()` doesn't need to take the source as parameter though, since it gets it from the prepper